.github/assert-git-changes.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python3
+import subprocess
+
+git_changes = subprocess.getoutput("git status --porcelain")
+
+print(git_changes)
+
+if git_changes != "":
+    print("Git changes detected! Failing")
```

Comment:
Maybe it's better to check explicitly for the updated files only inside the `lib/abi` dir?
Otherwise, it could be misunderstood in the future and may have compatibility issues.

Reply:
Agree. Fixed